src/board/system76/common/include/board/dgpu.h

```diff
@@ -1,10 +1,17 @@
 #ifndef _BOARD_DGPU_H
 #define _BOARD_DGPU_H
 
+#include <stdint.h>
+
 #ifndef HAVE_DGPU
     #define HAVE_DGPU 0
 #endif
 
+#if HAVE_DGPU
+    extern int16_t dgpu_temp;
+    extern uint8_t dgpu_duty;
```

Comment:
`dgpu_duty` needed? Everything else reads from `DCR4` directly.

Reply:
Technically not